tools/docker/image-build.sh

```diff
@@ -21,27 +21,31 @@ scriptdir="$(cd "${0%/*}"; pwd)"
 topdir="${scriptdir%/*/*/*}"
 
 usage() {
-    echo "usage: $(basename $0) [-hvn] <type>"
+    echo "usage: $(basename $0) [-hvbBR] <type>"
     echo "  mandatory:"
     echo "      type - image type <runner/builder>"
     echo "  options:"
     echo "      -h|--help - show this help menu"
     echo "      -v|--verbose - verbosity on"
-    echo "      -b|--base-image - Base OS image to use (Dockerfile 'FROM') "
+    echo "      -b|--base-image - Base OS image to use (Dockerfile 'FROM')"
+    echo "      -B|--builder-name - name for the builder image (default prplmesh-builder)"
```

Comment:
What is the use case for setting these names?

I would tend to prefer to use a tag instead, and always use the same tag for both. So `-t <tag>` as the option for the script, and `TAG=":$2"` with TAG defaulting to empty, and `--tag=prplmesh-builder$TAG` in the build call.